script/Arcueid/Day 3/03_00_ARC03_5_1.txt

```diff
@@ -397,10 +397,10 @@ But at that time, I didn't even think about enduring or suppressing those though
 -- そんなコトを当たり前のように考えて、実行していた。
 I was thinking and acting as though such impulses were totally natural.
 }
-[sha:1f82d4bbbd969608403d51a0b56c65f9ade73f69]{
+[offset:4737]{
 -- Page 271, Offset 4737.
 -- ……なら答えは簡単だ。
-In which case... the answer is obvious.
+Then... the answer is simple.
```

Comment:
That line is from an old file. Ross just changed this line yesterday. I'll revert it.

Reply:
Ah okay thanks!
I thought I fetched origin though...

Reply:
Hmm that's weird. I'm not sure where the problem came from then. Well unless you're editing from a notepad like me then it might be from that. :aichanteehee: